### PR TITLE
Feature/make default upload maxsize configurable

### PR DESCRIPTION
**Pull request by solidgoldpig**

### CODE REVIEW

File: lib/page/set-errors/set-errors.js [hunk 1]
```diff
@@ -91,41 +91,50 @@ const getInstanceErrorStrings = (controlInstance = {}, errorType) => {
 /*
 * @private
 */
-external.getFormattedError = (location, controlInstance, errorType, error, lang) => {
+external.getFormattedError = (position, controlInstance, errorType, error, lang) => {
```
Comment (elena-vi) [inline lib/page/set-errors/set-errors.js]: I'm curious to know why the param name change here

File: lib/page/set-multipart-form/set-multipart-form.unit.spec.js [hunk 2]
```diff
@@ -27,7 +28,7 @@ test('When setMultipartForm is passed a page containing upload controls', async
   })
 
   t.equal(pageInstance.encType, true, 'it should set page’s encType to true')
-  t.equal(pageInstance.components[0].maxSize, 10 * 1024 * 1024, 'it should set component’s maxSize')
+  t.equal(pageInstance.components[0].maxSize, setUploadControlsMaxSize.defaultMaxSize, 'it should set component’s maxSize')
```
Comment (elena-vi) [inline lib/page/set-multipart-form/set-multipart-form.unit.spec.js]: 👍 
`10 * 1024 * 1024` just looked super weird to me

File: lib/page/set-errors/set-errors.js (hunk 1 above)
Comment (solidgoldpig) [reply lib/page/set-errors/set-errors.js]: It just seemed a better description, that's all

File: lib/page/set-multipart-form/set-multipart-form.unit.spec.js (hunk 2 above)
Comment (solidgoldpig) [reply lib/page/set-multipart-form/set-multipart-form.unit.spec.js]: NB. this change isn't about it looking super weird, but decoupling it from whatever the default maxsize happens to be

**Pull request merged**